database/src/main/scala/cromwell/database/sql/tables/BackendKVStore.scala

```diff
@@ -0,0 +1,12 @@
+package cromwell.database.sql.tables
+
+case class BackendKVStore
```

Comment:
This is a bit of a misnomer. Better would be something like `BackendKVStoreEntry` or `BackendKVStoreRow`